src/string_utils.cpp

```diff
@@ -120,3 +120,22 @@ impl::LineSplitter::Iter impl::LineSplitter::Iter::operator++() noexcept {
     *this = Iter(rest_, false);
     return prev;
 }
+
+std::string lower_case(std::string_view str) {
```

Comment:
Hurrah!

Reply:
Yeah, why on earth does STL not already have this?